mantis/analysis/AnalysisSettings.py

```diff
@@ -69,6 +69,37 @@ def check_affine_transform(cls, v):
         return v
 
 
+class ConcatenateSettings(MyBaseModel):
+    concat_data_paths: list[str]
+    time_indices: Union[NonNegativeInt, list[NonNegativeInt], Literal["all"]] = "all"
```

Comment:
Negative integers are useful too:

```py
assert [1, 2, 3][slice(-2, -1)] == [2]
```

Reply:
Ah yes, for time it's easier. For the slicing... I think it will break my code because of the way we compute the slicing indeces
(i.e Z_slice.stop-Z_slice.start)

Reply:
What about:

```py
assert abs((-2) - (-1)) == 1
```